### PR TITLE
Fix #36364 ccleaner.com

### LINKED ISSUE #36364: ccleaner.com
### Issue URL (Annoyance)
[https://www.ccleaner.com/ru-ru/ccleaner](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.ccleaner.com%2Fru-ru%2Fccleaner)
### Comment
Username: @big-u-boss
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Ext
AdGuard version: | 3.2.1
Browser: | Chrome
Stealth mode options: | Hide your search queries, Remove X-Client-Data header from HTTP requests, Strip URLs from tracking parameters, Hide your Referrer from third-parties
Filters: | AdGuard Russian,<br>AdGuard Annoyances,<br>AdGuard Base,<br>AdGuard Tracking Protection,<br>AdGuard Social Media

### PR DESCRIPTION
**Pull request by tvinzz**
https://github.com/AdguardTeam/AdguardFilters/issues/36364

### CODE REVIEW

File: AnnoyancesFilter/sections/popups.txt [hunk 1]
```diff
@@ -81,6 +81,8 @@ sodapdf.com##.be-tooltip
 !
 ! Specific sites
 !
+ccleaner.com##.mfp-ready
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/popups.txt]: use #$# and check for useful popups

**Review by Alex-302 — CHANGES_REQUESTED** [review]

**Review by Alex-302 — APPROVED** [review]

File: RussianFilter/sections/specific.txt [hunk 2]
```diff
@@ -18,6 +18,7 @@ Ly9hbi55YW5kZXgucnUv
 ! adbetnet domains
 ||cdn.*/libs/b.js$third-party,domain=nnm-club.lib|nnm-club.me|nnm-club.tv|nnmclub.to|seasonvar.ru|mintmanga.com|realmadrid.ru
 !
+ccleaner.com##.trustpilot-widget
```
Comment (Alex-302) [inline RussianFilter/sections/specific.txt]: Not an ad

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: RussianFilter/sections/specific.txt (hunk 2 above)
Comment (slavaleleka) [reply RussianFilter/sections/specific.txt]: it is a third-party widget and now it's blocking by Tracking Protection. so maybe we should unblock it in Spyware filter 
`@@||googletagmanager.com/gtm.js$domain=ccleaner.com`
and block `##.trustpilot-widget` by Annoyance filter

Comment (Alex-302) [reply RussianFilter/sections/specific.txt]: @slavaleleka and `||widget.trustpilot.com^$third-party`

File: AnnoyancesFilter/sections/popups.txt [hunk 3]
```diff
@@ -81,8 +81,8 @@ sodapdf.com##.be-tooltip
 !
 ! Specific sites
 !
-ccleaner.com##.mfp-ready
-ccleaner.com#$#html { overflow: visible !important }
+ccleaner.com#$#.mfp-ready { display: none!important; }
+ccleaner.com#$#html { overflow: visible!important; }
 tr.link##.sweet-alert
 tr.link##.sweet-overlay
 retro-bit.com#$#.popup-newsletter { display: none!important }
```
Comment (krystian3w) [inline AnnoyancesFilter/sections/popups.txt]: gone `;`

**Review by krystian3w — CHANGES_REQUESTED** [review]

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**